src/content/jobs/full-stack-ai-lead.md

```diff
@@ -0,0 +1,114 @@
+---
+meta:
+    title: Fullstack AI Lead - Dołącz do Bethink!
+    description: Świetny developer dla każdego - starego i młodego
+    linkImageUrl: asdfasdf
```

Comment:
What about this? Same for heroImage

Reply:
Copied from existing job offer. It isn't used anywhere now. Don't want to clean it now